core/src/main/resources/lib/form/entry.jelly

```diff
@@ -89,8 +89,9 @@ THE SOFTWARE.
         <div class="setting-main">
           <d:invokeBody />
         </div>
-        <!-- used to display the form validation error -->
-        <div class="validation-error-area tr"><div colspan="2"></div><div></div><div></div></div>
+        <div class="validation-error-area">
+          <!-- Used to display the form validation status -->
+        </div>
```

Comment:
> LGTM, I've done a fair bit of manual testing and ATH and bom pass

ATH and BOM testing are [necessary but not sufficient](https://en.wikipedia.org/wiki/Necessity_and_sufficiency#Relationship_between_necessity_and_sufficiency) for a change of this scope. In order to fully ensure the change will not break existing consumers, we should ensure that there are no usages of APIs for which we are breaking compatibility.

The comment just a few lines above states:

> Needs to have a `tr` class otherwise custom uses of […] `findFollowingTR(listBox, "validation-error-area")` will break

This PR removes the `tr` class, which will cause usages of `findFollowingTR(listBox, "validation-error-area")` in plugins to break. I would advise scanning the plugin corpus looking for such usages, and if any are found in non-deprecated plugins, either:

- Deprecating the plugin in the Update Center, linking to this PR in the deprecation notice
- Filing a PR to make the plugin compatible with this PR and making a good faith effort to have the PR merged and released in a timely fashion

An example of this is `git-parameter-plugin`, which as of this PR now breaks with the following stack trace:

```
TypeError: findFollowingTR(...).firstChild is null
    gitParameterUpdateSelect http://127.0.0.1:8080/adjuncts/eb9dda3e/net/uaznia/lukanus/hudson/plugins/gitparameter/javascript/git-parameter-select.js:8
    <anonymous> http://127.0.0.1:8080/adjuncts/eb9dda3e/net/uaznia/lukanus/hudson/plugins/gitparameter/javascript/git-parameter-select.js:96
    h http://127.0.0.1:8080/static/eb9dda3e/scripts/hudson-behavior.js:1568
    refillOnChange http://127.0.0.1:8080/static/eb9dda3e/scripts/hudson-behavior.js:1583
    <anonymous> http://127.0.0.1:8080/adjuncts/eb9dda3e/net/uaznia/lukanus/hudson/plugins/gitparameter/javascript/git-parameter-select.js:94
    applySubtree http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:116
    applySubtree http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:111
    applySubtree http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:93
    apply http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:76
    start http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:71
    onload http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:133
    onload http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:133
    addLoadEvent http://127.0.0.1:8080/static/eb9dda3e/scripts/behavior.js:132
    createSearchBox http://127.0.0.1:8080/static/eb9dda3e/scripts/hudson-behavior.js:2049
    <anonymous> http://127.0.0.1:8080/job/freestyle/build?delay=0sec:9
behavior.js:119:31
```

Please note that although this is one example of a problem, it is not necessarily the only example. To make a change like this with confidence, the corpus of plugins should be scanned for usages of the relevant APIs and the results systematically examined. This observation applies not only to this PR but to any PR of this scope.

Reply:
Hey! Thanks for this.

Do you have any reproduction steps for the `git-parameter-plugin` breakage? I can't seem to get the error on my end.

I'll investigate the others and compile a list to fix or deprecate.

Thanks :)

Reply:
I had never used this plugin before but I suspected that if I hit the `findFollowingTR` function that I would see a problem. Reading the code, it seemed to be executed when filling in parameters during a build, so I created a parameterized Freestyle job with a Git branch parameter and then tried to run that job. I got the error when I started typing stuff into the Git branch parameter field.

Reply:
Reproduced it and fixed it https://github.com/jenkinsci/git-parameter-plugin/pull/94 - thanks Basil!

Using this search https://github.com/search?q=org%3Ajenkinsci+findFollowingTR%28*%2C+%22validation-error-area%22%29&type=code it seems the only remaining plugin is https://github.com/jenkinsci/bmc-rpd-plugin, which hasn't been updated in six years, happy to update it or deprecate it if need be.

Thanks! :)

Reply:
It’s basically unused and has an active security warning so we can leave that one

Reply:
> Reproduced it and fixed it jenkinsci/git-parameter-plugin#94 - thanks Basil!

Great! Once that PR is merged and released the 59,936 users of Git Parameter can be assured of a smooth upgrade experience without regression.

> It’s basically unused and has an active security warning so we can leave that one

I agree that we can leave behind BMC RPD Plugin. In order to leave this plugin behind in a way that retains the trust of our users, one of whom [recently wrote](https://www.sghill.net/2022/de-risking-rollouts/) "the community does a wonderful job of being clear about what is not supported, and supporting well what is supported," I think we should add that plugin to the [deprecations list](https://github.com/jenkins-infra/update-center2/blob/master/resources/deprecations.properties). That way, it is clear to users that plain text credential storage is not the only reason that this plugin is not recommended.

Reply:
Opened a PR for that here - https://github.com/jenkins-infra/update-center2/pull/597 - hope I've done it correctly :) Cheers